snaps/concepts/design-guidelines.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Snaps design guidelines
+description: Guiding principles for designers, developers, builders, and writers to create snap install flows that are accessible for all types of users.
+---
+
+# Snaps design guidelines
+
+:::note Summary
+Outlined here are guiding principles for designers, developers, builders, and writers to create snap install flows that are accessible for all types of users. You should use the guidelines in this document when introducing your snap within a dapp or website.
+:::
+
+## Why this matters
+
+The snap installation process contains critical info about your snap, including what it does, how it enhances your application, and why it’s beneficial for users. It's important to provide this information on your website or dapp to help users understand the purpose and benefits of the snap before installing it. Without this information, users may drop out during installation or install the snap without fully understanding its purpose.
+
+### Think like your users, write like a human
+
+Consider whether the details that interest you as a developer are relevant to the user’s experience. Often times, content can be drastically reduced by cutting jargon and run-on-sentences. Try reading your content out loud to hear what stands out most in your messaging. If something is hard to say, it’s probably hard to read.
+
+- **Be clear**
+  When labeling buttons or actions, use descriptive action verbs instead of vague phrases. _Install_ is more clear than _Complete_, for example.
+
+- **Be concise**
+  Use short, simple words. Make every word earn its place on the screen.
+
+- **Be consistent**
```

Comment:
```suggestion
Consider whether the details that interest you as a developer are relevant to the user’s experience. Often times, content can be drastically reduced by cutting jargon and run-on sentences. Read your content out loud to hear what stands out most in your messaging. If something is hard to say, it’s probably hard to read.

- **Be clear.**
  When labeling buttons or actions, use descriptive action verbs instead of vague phrases. _Install_ is more clear than _Complete_, for example.

- **Be concise.**
  Use short, simple words. Make every word earn its place on the screen.

- **Be consistent.**
```